src/lib/components/workflow/filter-search/boolean-filter.svelte

```diff
@@ -23,7 +23,7 @@
 </script>
 
 <MenuContainer>
-  <MenuButton id="boolean-filter" controls="boolean-filter-menu">
+  <MenuButton unroundLeft class="!border-l-0" id="boolean-filter" controls="boolean-filter-menu">
```

Comment:
What's going on with the `important`?

Reply:
By default our inputs, buttons, menu buttons, etc. have borders and border radius, however there are instances where they are combined with other elements, and we want to remove that border and border-radius. I don't like the pattern of overriding default component styles by adding a css class, I would rather it be controlled by a prop so as to maintain visual consistency. That's exactly why I went with [this approach](https://github.com/temporalio/ui/pull/1589#discussion_r1311932741) for `<Button />`, I just haven't done it for `<MenuButton />`.

I'm not sure if important is necessary here though, we may just be able to pass the regular `border-l-0` tailwind class, @laurakwhit did you happen to test that out?

Reply:
I think it would be better to use the `group` or `peer` classes in Tailwind.

Reply:
For example, it looks like we _know_ this is the first of its kind the group. Our styling could have affordances for this _without_ us needed to specify it.

Reply:
> I'm not sure if important is necessary here though, we may just be able to pass the regular border-l-0 tailwind class, @laurakwhit did you happen to test that out?

I did not. I just pulled that from somewhere similar.. Maybe `ConditonalMenu`?